Library/ViewModels/PaymentMethodSettingsViewModel.swift

```diff
@@ -159,10 +159,6 @@ public final class PaymentMethodSettingsViewModel: PaymentMethodsViewModelType,
             configuration.merchantDisplayName = Strings.general_accessibility_kickstarter()
             configuration.allowsDelayedPaymentMethods = true
 
-            if featureStripeLinkEnabled() {
```

Comment:
Per @tedPen, we don't need to show Stripe Link from the payment method settings screen - only in the pledge flow. Removing this for clarity's sake, since it would never show anyways.